### PR TITLE
Improve Sphinx docs highlighting using RST markup

### PR DESCRIPTION
**Pull request by disrupted**
we can use reStructuredText markup to better highlight code blocks in pydocs

### CODE REVIEW

**Review by raminqaf — APPROVED** [review]
🆒

File: kpops/cli/exception.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 class ClassNotFoundError(Exception):
-    """Similar to builtin ModuleNotFoundError; class doesn't exist inside module."""
```
Comment (sujuka99) [inline kpops/cli/exception.py:2]: How did you choose where to use single or double back tick?

Comment (disrupted) [reply kpops/cli/exception.py:2]: that's one of the things I am quite unsure about, I usually just reused what was there already. but noticed that there's both ways throughout the codebase. do you know if there's a difference?

Comment (sujuka99) [reply kpops/cli/exception.py:2]: [There is](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup), but I am not sure if it matter at all in our case

Comment (disrupted) [reply kpops/cli/exception.py:2]: I checked the highlighting in nvim but for me there's no difference between single and double backtick

**Review by sujuka99 — APPROVED** [review]

**Pull request merged**